packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderButton/ConditionBuilderButton.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import cx from 'classnames';
+
+import PropTypes from 'prop-types';
+import { Tooltip } from '@carbon/react';
+import { pkg } from '../../../settings';
+
+const blockClass = `${pkg.prefix}--condition-builder`;
+
+export const ConditionBuilderButton = ({
+  className,
+  label,
+  hideLabel = false,
+  tooltipAlign = 'top',
+  renderIcon: Icon,
+  onClick,
+  showToolTip,
+  role,
+  ...rest
+}) => {
+  const button = (
```

Comment:
as a convention we should use the component declaration notation

```
const Button = () => { ... }
...
<Button />
```

Reply:
modified as advised